src/js/homepage.js

```diff
@@ -0,0 +1,147 @@
+import React from "react";
+import ReactDOM from "react-dom";
+import _ from "lodash";
+
+function UpdateListButton(props) {
+  return React.createElement('button',
+    {
+      type: 'button',
+      onClick: props.onClick,
+      className: props.button.className,
+    },
+    props.button.text);
+}
+
+class Handle extends React.Component {
+  render() {
+    if(this.props.isHomeTimeline) {
+      return React.createElement('div', {className: 'handle'}, `@${this.props.twitterHandle}`);
+    } else {
+      return null;
+    }
+  }
+}
+
+class Avatar extends React.Component {
+  render() {
+    let user = this.props.user;
+    return React.createElement('div', {className: 'userInfo'},
+      React.createElement('div', {className: 'img'},
+        React.createElement('img', {src: `${user.profileImageUrl}`, className: 'img-circle'})),
+      React.createElement('div', {className: 'name'}, `${user.name}`),
+      React.createElement(Handle, {twitterHandle: user.twitterHandle, isHomeTimeline: this.props.isHomeTimeline})
```

Comment:
this can be cleaned up versus letting the component render or not render within itself. Let's do that here